test/cctest/test_node_debug_support.cc

```diff
@@ -0,0 +1,127 @@
+/* {{debug_symbols_file}} */
+
+/*
+ * This file is a template used by ./tools/gen-postmortem-metadata.py to
+ * generate a proper test file
+ */
+
+#include "gtest/gtest.h"
+#include "node_test_fixture.h"
+#include "node.h"
+#include "node_internals.h"
+#include "udp_wrap.h"
+#include "v8.h"
+#include "req_wrap-inl.h"
+#include "tracing/agent.h"
+
+const int node::Environment::kContextEmbedderDataIndex;
+
+
+class DebugSymbolsTest : public EnvironmentTestFixture {
+};
+
+
+class TestHandleWrap : public node::HandleWrap {
+  public:
+   size_t self_size() const override { return sizeof(*this); }
+
+   TestHandleWrap(node::Environment* env, v8::Local<v8::Object> object, uv_handle_t* handle) :
+     node::HandleWrap(env, object, handle, node::AsyncWrap::PROVIDER_TIMERWRAP) {}
+};
+
+
+class TestReqWrap : public node::ReqWrap<uv_req_t> {
+  public:
+   size_t self_size() const override { return sizeof(*this); }
+
+   TestReqWrap(node::Environment* env, v8::Local<v8::Object> object) :
+     node::ReqWrap<uv_req_t>(env, object, node::AsyncWrap::PROVIDER_TIMERWRAP) {}
+};
+
+TEST_F(DebugSymbolsTest, ContextEmbedderDataIndex) {
+  EXPECT_EQ(nodedbg_environment_context_idx_embedder_data, node::Environment::kContextEmbedderDataIndex);
```

Comment:
Long line.  Can you run `make lint`?  I'll stop pointing out style issues from here on.

Reply:
Sorry, I forgot to run `make lint` on my last commit. I'm running it and a spell-checker on comments right now, will update the PR soon.

Reply:
Out of curiosity, there are ~20 Python scripts on `tools/`, wouldn't it be good to have a Python linter as well? I can make a PR with this if there's interest in it